.travis.yml

```diff
@@ -6,18 +6,18 @@ env:
 matrix:
   fast_finish: true
   include:
-    - env: DM=~4.2
+    - env: DM=~6.0
       php: 5.3
-    - env: DM=~4.2
+    - env: DM=@dev
```

Comment:
I'd have put the @dev with a latest PHP version, so this does not break when changing DM master to require PHP 5.5+